docs/changelog.md

```diff
@@ -12,14 +12,27 @@ versioning](http://semver.org) policy.
 Changes will be added here periodically from the "Suggested changelog
 entry" block in pull request descriptions.
 
-## 3.0.0 RC 1
+## 3.0.0 RC 1 (May 21, 2025)
 
-We may add one more opt-in feature (embedded subinterperters) before the
-final release.
+Since this is a large release, we are providing a release candidate to give
+projects time to test! We also now provide
+[SPEC 4](https://scientific-python.org/specs/spec-0004/) nightly wheels.
+
+Pybind11 3.0 includes an ABI bump, the first required bump in many years
+(except on Windows). This release contains the smart-holder branch, multi-phase
+init and subinterpreter support, `py::native_enum`, an interface to warnings,
+typing improvements, and more. CMake now defaults to FindPython mode. Please
+check our upgrade guide for more info on upgrading!
+
+Support for Python 3.14, 3.14t, GraalPy, and PyPy 3.11 has been added, while
+legacy support for Python 3.7, PyPy 3.8/3.9, and CMake \<3.15 has been removed.
+Most deprecated features have been kept for this release, but anything
+producing a warning in 3.0 may be removed in a future 3.x version. We also now
+have a deprecation page.
 
 New Features:
 
-- The `smart-holder` branch has been merged, enabling
+- The `smart_holder` branch has been merged, enabling
```

Comment:
Maybe keep the underscore? Because that was the actual branch name.

Reply:
Meaning keep the change?

Reply:
Oops, yes, sorry I was going too fast last night.